docs/sources/administration/team-management/configure-grafana-teams.md

```diff
@@ -0,0 +1,213 @@
+---
+aliases:
+  - 
+description: Learn how to configure Grafana Teams
+labels:
+  products:
+    - enterprise
+    - oss
+    - cloud
+keywords:
+  - authorization
+  - microservices
+  - architecture
+menuTitle: Configure  Grafana Teams
+title: Configure Grafana Teams
+weight: 200
+---
+
+# Configure Grafana Teams
+
+This topic describes how to set up and configure Grafana Teams.
+
+For a tutorial on working with Teams see [Create users and teams](https://grafana.com/tutorials/create-users-and-teams/).
```

Comment:
https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/#links-and-references

```suggestion
For a tutorial on working with Teams, refer to [Create users and teams](https://grafana.com/tutorials/create-users-and-teams/).
```